base/wrappers/data_object_enricher.go

```diff
@@ -26,19 +28,26 @@ type DataObjectEnricherI interface {
 
 	// Close allows the plugin to close any connections and make sure that all data objects still in the buffer are handled and written to the DataObjectWriter.
 	// The first return parameter is the number of data objects that were actually enriched.
+	// Deprecated: use a DataObjectEnricherFactoryFn with correct cleanup function instead.
 	Close(ctx context.Context) (int, error)
```

Comment:
Throw it away immediately. There are no active implementations of the enrichers yet at the moment